### ISSUE TITLE
Are the image files included in the four data sets different image files

### ISSUE THREAD

**Issue by Tokyo81** [X-PLUG/mPLUG-DocOwl#40]
DocStruct4M
DocDownstream-1.0
DocReason25K
DocLocal4K
 
Are the image files included in the four data sets different image files

**Comment by HAWLYQ** [comment]
Yes. Some images may be shared, but I suggest you to download images independently.
